data/names.txt

```diff
@@ -216,6 +224,17 @@ phrase "singular nouns"
 		"Banner"
 		"Voyage"
 		"Rose"
+		"Song"
+		"Hawk"
+		"Raven"
```

Comment:
Yeah, and I believe those have been there for a long time. I'll get rid of Hawk in the civilian name list though.